core/block/editor/dashboard.go

```diff
@@ -75,6 +78,40 @@ func (p *Dashboard) StateMigrations() migration.Migrations {
 	}})
 }
 
+// autoInstallBinWidget installs bin widget for the existing users if it was not added/removed before and the user has archived objects
```

Comment:
misleading description

Reply:
removed, cause method needs not be done on another level